src/module.ts

```diff
@@ -1,27 +1,30 @@
-import { PanelPlugin, FieldOverrideContext, FieldConfigProperty, FieldColorModeId } from '@grafana/data';
-import { PanelOptions, FieldOption } from './types';
+import {
+  PanelPlugin,
+  FieldOverrideContext,
+  FieldConfigProperty,
+  FieldColorModeId,
+  SelectableValue,
+  SelectFieldConfigSettings,
+} from '@grafana/data';
+import { getFieldTypeIcon } from '@grafana/ui';
+import { PanelOptions } from './types';
 import { PlotlyPanel } from './PlotlyPanel';
-import { MultiSelectValueEditor } from './MultiSelect';
 
 export const plugin = new PanelPlugin<PanelOptions>(PlotlyPanel)
   .setPanelOptions((builder) => {
     return builder
-      .addSelect({
+      .addFieldNamePicker({
```

Comment:
This replaces a basic dropdown editor with an editor that's specifically designed for selecting fields out of a data frame (new to Grafana 8/9).